apps/client/src/pages/index.tsx

```diff
@@ -0,0 +1,6 @@
+export * from './loading';
+export * from './home';
+export * from './room';
+export * from './race';
+export * from './leaderboard';
+export * from './layout';
```

Comment:
Can you please order the exports in alphabetical order?

Reply:
added eslint plugin to auto sort exports f25f9c09834f80362b05129674caec64acebb9ad